### PR TITLE
add `()` to calls just after a closure expression

### PR DESCRIPTION
**Pull request by koher**
Fix the probem #6.

### CODE REVIEW

File: Sources/TranslateExpressions.swift [hunk 1]
```diff
@@ -25,7 +25,14 @@ extension FunctionCallExpression {
...
             hasNew = false
         }
-        return "\(hasNew ? "new " : "")\(expression.javaScript(with: indentLevel))(\(jsArguments.joined(separator: ", ")))"
+        let jsExpression: String = "\(hasNew ? "new " : "")\(expression.javaScript(with: indentLevel))"
+        let jsParenthesizedExpression: String
+        if expression is ClosureExpression {
+            jsParenthesizedExpression = "(\(jsExpression))"
+        } else {
+            jsParenthesizedExpression = jsExpression
+        }
```
Comment (rintaro) [inline Sources/TranslateExpressions.swift]: I prefer
```swift
var jsExpression: String = ...
if ... {
  jsExpression = "(\(jsExpression))"
}
```
style. But, okay.

**Review by rintaro — APPROVED** [review]
LGTM!

**Comment by koher** [comment]
@rintaro I updated the codes as you proposed. Please check it.

**Review by rintaro — APPROVED** [review]
LGTM

**Review by masarusanjp — APPROVED** [review]
LGTM!

**Pull request merged**